vllm/executor/executor_base.py

```diff
@@ -137,3 +183,143 @@ async def check_health_async(self) -> None:
         """Checks if the executor is healthy. If not, it should raise an
         exception."""
         self.check_health()
+
+
+class DistributedExecutorBase(ExecutorBase):
```

Comment:
Given that `DistributedExecutorBase` is used in V0 but not in V1, do you envision this going away when we transition to V1 entirely?

Reply:
yes.